services/github/src/github-auth.js

```diff
@@ -12,10 +12,10 @@ const retryPlugin = (octokit, options) => {
       try {
         return await request(options);
       } catch (err) {
-        if (attempt === retries || err.name !== 'HttpError' || err.status !== 404) {
+        if (attempt === retries || err.name !== 'HttpError' || err.status < 404) {
```

Comment:
I think we should retry 404 and >=500.  Most 4xx's shouldn't be retried.  For example, 409 conflict is a permanent error.

Reply:
oh ok